README

```diff
@@ -91,9 +102,3 @@ if you want to fix them.
 
 Archive file names are not re-encoded to current locale. You can use `iconv`
 for that.
-
-Archive file names are truncated to 100 characters (the limit for original tar
```

Comment:
This subtlety got past me.  I've restored your original _known problem_ expanding it to describe how it handles the GNU long file name extension.  Addressed in commit: 40b6193be24af863a930163edb612be0b6026d1d